thin-replica-server/include/thin-replica-server/thin_replica_impl.hpp

```diff
@@ -286,6 +291,7 @@ class ThinReplicaImpl {
     metrics.setAggregator(aggregator_);
     uint16_t update_aggregator_counter = 0;
     metrics.subscriber_list_size.Get().Set(config_->subscriber_list.Size());
+    metrics.last_sent_block_id.Get().Set(last_sent_block_id_);
```

Comment:
Note, the metric in this case may not monotonically increase. Consider a scenario where TRS sent 100 blocks to a client. The client only received 70 blocks (perhaps there was no agreement on the 71st block), the client would resubscribe for 71st block, so the metric would change from 100 -> 71 when the 71st block is sent by the replica. Just an fyi.